cmp/form/field/BaseFieldModel.js

```diff
@@ -48,9 +46,15 @@ export class BaseFieldModel {
      */
     @observable validationDisplayed = false;
 
+    //----------------------
+    // Implementation State
+    //----------------------
     @observable _disabled;
     @observable _readonly;
 
+    // An array with the result of evaluating each rule (null for not complete, or list of errors)
```

Comment:
Could we clarify the contents here a bit more?  `null` if the rule has not run, a `string` error if there's a problem, but what kind of element if the rule passes OK?  Empty string?  (If that's the case seems a bit weird TBH, although since it's not the public API I guess that's fine.)